protocol/SignalingProtocol.proto

```diff
@@ -248,8 +248,13 @@ message SdkMetric {
         VIDEO_DISCARDED_PPS = 47;
         VIDEO_PLIS_SENT = 48;
         VIDEO_RECEIVED_JITTER_MS = 49;
+        VIDEO_ENCODE_HEIGHT = 64;
```

Comment:
Confirming with Tianyu, actually he need the video frame height and width related metrics for another project, so it would be great if we send them back to tincan.

Reply:
Does that mean we need to make the changes to Media server first?

Reply:
yes, @XHatan is working on the changes to Media.